index.js

```diff
@@ -103,7 +103,36 @@ var writerOpts = {
     return commit;
   },
   groupBy: 'type',
-  commitGroupsSort: 'title',
+  commitGroupsSort: function(group1, group2) {
```

Comment:
Can't think of an easier way to bubble up things in order of:
Breaking, Features, Bug, {{title}}